3ds/libsidplay/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: badda71 <me@badda.de>
+
+pkgname=3ds-libsidplay
+pkgver=1.36.60
```

Comment:
Most distros are using 1.36.59. What's the provenance of this one?

Reply:
done

Reply:
Release notes say:
CHANGES since 1.36.59:

* Apply minor patches to silence warnings about potentially uninitialized
  values.
* Get rid of references to the old project home at GeoCities, which
  has been removed with the shut-down of GeoCities and won't reappear
  elsewhere.
* Patch ios::binary check for GCC 4.3.0 C++.